FreeRTOS-Plus/Source/FreeRTOS-Plus-CLI/FreeRTOS_CLI.c

```diff
@@ -127,6 +130,7 @@ BaseType_t xReturn = pdFAIL;
 			pxNext has nowhere to point. */
 			pxNewListItem->pxNext = NULL;
 
+			static CLI_Definition_List_Item_t *pxLastCommandInList = &xRegisteredCommands;

```

Comment:
Do you have a reason to move this line down here, instead of declaring it in the beginning of this function?